oid4vc/sd_jwt_vc/cred_processor.py

```diff
@@ -74,7 +74,7 @@ async def issue(
         assert supported.format_data
         assert supported.vc_additional_data
 
-        sd_list = supported.vc_additional_data.get("sd_list", [])
+        sd_list = supported.vc_additional_data.get("sd_list") or []
```

Comment:
Interesting. Thought this would work.

Reply:
Indeed; subtle nuance is that `.get` will return a None if `sd_list` is present and null. We always want to treat it as a list, even if a null is given, so we have to do the `or` syntax here.